clinic/clinic-user-guide-for-tiup.md

```diff
@@ -0,0 +1,348 @@
+- Meta-information for the cluster itself: in `meta.yaml` (this file is located at the top level of the directory that stores collected data)
+- Monitoring data: in the `/monitor` file directory. The monitoring data is compressed by default and cannot be viewed directly. To directly view the JSON files with monitoring data, disable compression with the `--compress-metrics=false` parameter when collecting data.
+
+### Step 4. Upload data
+
+To provide cluster diagnostic data to PingCAP technical support staff, you need to upload the data to the Clinic Server first, and then send the obtained data access link to the staff. The Clinic Server is a cloud service that stores and shares diagnostic data securely.
+
+Depending on the network connection of the cluster, you can choose one of the following methods to upload data:
+
+- Methods 1: if the network where the cluster is located can access the internet, you can [directly upload data using the upload command](#method-1-upload-directly).
+- Methods 2: if the network where the cluster is located cannot access the internet, you need to [pack the data and then upload it](#method-2-pack-and-upload-data).
+
+#### Method 1. Upload directly
+
+If the network where the cluster is located can access the internet, you can directly upload the folder with collected data obtained in [Step 2: Collect data](#step-2-collect-data) using the following command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+ tiup diag upload ${filepath} -u=username -p='password'
+```
+
+> **Note:**
+>
+> TiDB Clinic is currently in the Beta testing stage for invited users only. If you need to upload data to the Clinic Server using Diag, contact [PingCAP technical support](https://en.pingcap.com/contact-us/) to get a trial account first.
+
+The following is an example output:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+[root@Copy-of-VM-EE-CentOS76-v1 qiaodan]# tiup diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Enter Username: username
+Enter Password: >>>>>>>>>>>>>>>>>>>>>>>>>>>>>><>>>>>>>>>
+Completed!
+Download URL: "https://clinic.pingcap.com:4433/diag/files?uuid=XXXX"
+```
+
+After the upload is complete, you need to send the data access link of `Download URL` to the PingCAP technical support staff you contacted before.
+
+> **Note:**
+>
+> Currently, only PingCAP technical support staff can open the data access link. External users temporary **cannot** open the link.
+
+#### Method 2. Pack and upload data
+
+If the network where your cluster is located cannot access the internet, you need to pack the data on your intranet and upload the data package to the Clinic Server using a device with internet access. The detailed operations are as follows:
+
+1. Pack collected in [Step 2. Collect data](#step-2-collect-data) and compress and encrypt the data package by running the following command:
```

Comment:
```suggestion
1. Pack the collected data obtained in [Step 2. Collect data](#step-2-collect-data) by running the following command:
```

Reply:
line 225 already tells that Diag encrypts and compresses the data during packaging